.travis.yml

```diff
@@ -1,14 +1,15 @@
 dist: trusty
 sudo: required
 language: go
-go: "1.12.x"
+go: "1.14.x"
 
 env:
-  - GO111MODULE=on CONTAINERD_VERSION=1.1.5 RUNSC_VERSION=2019-08-06 TEST=untrusted-workload
-  - GO111MODULE=on CONTAINERD_VERSION=1.2.1 RUNSC_VERSION=2019-08-06 TEST=untrusted-workload
-  - GO111MODULE=on CONTAINERD_VERSION=1.2.1 RUNSC_VERSION=2019-08-06 TEST=runtime-handler
-  - GO111MODULE=on CONTAINERD_VERSION=1.2.1 RUNSC_VERSION=2019-08-06 TEST=runtime-handler-shim-v2
-
+  - CONTAINERD_VERSION=1.1.5 RUNSC_VERSION=release-20200219.0 TEST=untrusted-workload
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=untrusted-workload
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler-shim-v2
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler-shim-v2
+  - CONTAINERD_VERSION=1.3.3 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler-shim-v2
```

Comment:
I'm probably just missing information here, but why only run the `runtime-handler-shim-v2` test and not `runtime-handler` or `untrusted-workload`?

Reply:
I added shim v1 (runtime-handler). I think that's what I meant to do in the first place.
The untrusted workload annotation isn't supported anymore.